### PR TITLE
SharkNetSniffer

### PR DESCRIPTION
**Pull request by BuffBaby253**
A payload for the Sharkjack that captures network traffic and saves it in the loot folder

### CODE REVIEW

File: payloads/library/Wireshark PCAP Capture/payload.sh
Comment (hak5peaks) [inline payloads/library/Wireshark PCAP Capture/payload.sh:1]: This is in the wrong directory. Please move your payload to a category. `payloads/library/**category**/Wireshark_PCAP_Capture/payload.sh` Also remove spaces in the directory name, replacing them with `-` or `_`

**Review by hak5peaks — CHANGES_REQUESTED** [review]

File: payloads/library/Wireshark PCAP Capture/payload.sh
Comment (BuffBaby253) [reply payloads/library/Wireshark PCAP Capture/payload.sh:1]: will do, thank you

File: payloads/library/recon/Shark-Net-Sniff/payload.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+NETMODE DHCP_CLIENT
+while [ -z "$IPADDR" ]; do sleep 1 && IPADDR=$(ifconfig eth0 | grep "inet addr"); done
+
+LED ATTACK
+
+# using tcpdump to capture network traffic and save to loot directory
+tcpdump -i $INTERFACE -w $LOOT_DIR/net-traffic_$COUNT.txt &
+
+# sleep command will let it run for 1 minute, or you can edit to run however much you want
+sleep 60
```
Comment (hak5peaks) [inline payloads/library/recon/Shark-Net-Sniff/payload.sh:38]: I would add a variable for sleep at the top of the payload. This makes it easier for the user to change the value. 
```
seconds=60
sleep $seconds
```

**Review by hak5peaks — CHANGES_REQUESTED** [review]

**Review by hak5peaks — CHANGES_REQUESTED** [review]

**Review by BuffBaby253 — COMMENTED** [review]
right that's much better thank you

**Pull request merged**